packages/zero-cache/src/services/view-syncer/client-handler.ts

```diff
@@ -289,6 +308,16 @@ function makeRowPatch(patch: RowPatch): RowPatchOp {
   }
 }
 
+export function rest(id: RowKey, value: Row) {
+  const rest: Record<string, Value> = {};
+  for (const key in value) {
```

Comment:
Use `for (const key of Object.keys(value))` so it only checks the current object and not also the prototype.